### PR TITLE
Update FOSSA action

### PR DESCRIPTION
**Pull request by gab-arrobo**

### CODE REVIEW

**Review by thakurajayL — APPROVED** [review]

**Comment by thakurajayL** [comment]
Fossa still fails.

**Comment by gab-arrobo** [comment]
> Fossa still fails.

Yes, FOSSA will keep failing until @dafATonf help us configuring the new token as a secret in the `omec-project`. So, I need to wait for him to help us with this before I open PRs in the other repos replacing/updating the FOSSA action

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -46,9 +46,9 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: FOSSA scan
-        uses: fossa-contrib/fossa-action@v3
+        uses: fossas/fossa-action@main
         with:
-          fossa-api-key: 9dc8fa92e3dd565687317beb87b56d89
+          api-key: ${{secrets.fossaApiKey}}
```
Comment (gab-arrobo) [inline .github/workflows/main.yml:51]:
```suggestion
          api-key: ${{secrets.FOSSA_OMEC_ORG_PO}}
```

**Review by thakurajayL — APPROVED** [review]

**Comment by dafATonf** [comment]
Gabriel -- Moat uawa of github secrets that I've seen use ${{
secrets.FOSSA_OMEC_ORG_PO }}.  The variable is surrounded by spaces.   I am
unable to find an explicit definition that it needs to be done, but worth a
try.   I'll keep looking to see if it something else comes to mind.

-daf

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -46,9 +46,9 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: FOSSA scan
-        uses: fossa-contrib/fossa-action@v3
+        uses: fossas/fossa-action@main
         with:
-          fossa-api-key: 9dc8fa92e3dd565687317beb87b56d89
+          api-key: ${{secrets.FOSSA_OMEC_ORG_PO}}
```
Comment (gab-arrobo) [inline .github/workflows/main.yml:51]:
```suggestion
          api-key: ${{ secrets.FOSSA_OMEC_ORG_PO }}
```

**Comment by gab-arrobo** [comment]
> Gabriel -- Moat uawa of github secrets that I've seen use ${{ secrets.FOSSA_OMEC_ORG_PO }}. The variable is surrounded by spaces. I am unable to find an explicit definition that it needs to be done, but worth a try. I'll keep looking to see if it something else comes to mind.
> -daf

@dafATonf, the spaces do not seems to be the issue because I added the spaces and I still get the same error

**Pull request merged**